torch/nested/__init__.py

```diff
@@ -0,0 +1,53 @@
+import os
+import torch
+
+from . import nested
+from . import codegen
+
+USE_NESTEDTENSOR = os.getenv('USE_NESTEDTENSOR', 'OFF') == 'ON'
+if not USE_NESTEDTENSOR:
+    raise RuntimeError("Attempting to use NestedTensor code "
+                       "without the environment flag USE_NESTEDTENSOR "
+                       "set to ON")
+
+NestedTensor = nested.NestedTensor
+
+def _nary_gen(out_dtype=None):
+    # Follows signature of torch nary functions
+    def _nary(*args, **kwargs):
```

Comment:
You can write `_nary(func_name, func, *inputs, out=None, **kwargs)`

Reply:
Not in Python 2 :)